Tools/requiredPackages.R

```diff
@@ -0,0 +1,65 @@
+args <- commandArgs(trailingOnly = TRUE)
+
+if (length(args) == 0) {
+  stop("\nRequires: path to R folder")
+} else if (length(args) == 1) {
+  lib <- args[1]
+} else {
+  stop("\nAccepts only one argument")
+}
+
+if (dir.exists(lib)) {
+  if (any(endsWith(list.files(lib), ".R"))) {
+    setwd(lib)
+  } else {
+    stop("Could not find any R files in the directory")
+  }
+} else {
+  stop("Could not find directory")
+}
+
+options("repos" = "https://cloud.r-project.org")
+
+if (! "stringr" %in% installed.packages()) {
+  install.packages("stringr")
+}
+library(stringr)
+
+basePkgs <- installed.packages(priority="high")
+basePkgs <- basePkgs[basePkgs[, "Priority"] == "base", 1]
+
+reqPkgs <- NULL
+expr <- '([a-zA-Z0-9.]{2,}(?<![.]))(?:::|:::)[a-zA-Z0-9._]+'
```

Comment:
@TimKDJ can you please add a comment for this regex.

Reply:
# matches namespace in [valid namespace -> triple or double colons -> valid function name]
@FransMeerhoff

Reply:
@TimKDJ @akashrajkn 

- Added GPArotation to the list
- Added comment to RegExp
- Updated the reduced Framework in https://static.jasp-stats.org/development/R3.3%20OSX%20JASP%200.8.3.zip 
(Window follows)

Reply:
I think it's best to add GPArotation in the script (as per my post), so we do not forget it at some point.